src/resources/views/crud/columns/color.blade.php

```diff
@@ -0,0 +1,30 @@
+{{-- regular object attribute --}}
```

Comment:
I see this column now spans across two rows. That's too much. Ideally, all columns should have its values on one line alone. That would allow the dev to include it in a table without making the table too wide or too tall.

So instead of this:
<img width="608" alt="CleanShot 2022-10-11 at 11 12 26@2x" src="https://user-images.githubusercontent.com/1032474/195035426-9c732286-6650-4608-a730-3ddd83676ee3.png">

Let's please limit the color preview to a square. Not a long rectangle... just a square. Same height, same length. That will allow us to have the color preview and the color hex on the same line.

Alternatives to square: circle or squircle (actually I think circle would probably look the best... try things out and choose one you think works best)

Reply:
Cosmetic UI changes applied as suggested.